salt/dns/init.sls

```diff
@@ -1,14 +1,16 @@
-boto-pkgs:
-  pkg.installed:
-    - pkgs:
-      - python3-boto
-      - python3-boto3
+boto:
+  pip.installed:
+    - pip_bin: /usr/bin/salt-pip
+
+boto3:
+  pip.installed:
+    - pip_bin: /usr/bin/salt-pip
 
 {% set public_ipv4 = salt["mine.get"]("*", "public_ipv4") %}
 
 # We assume that a server will always have an IPv4 address.
 
-# TODO: Update back to boto3_route53 when https://github.com/saltstack/salt/pull/60951 makes it into a release
+# TODO(@JacobCoffee): Update back to boto3_route53
```

Comment:
I don't recall why boto3 is preferred, but this is the diff that originally moved us back to just both: https://github.com/python/psf-salt/pull/249/commits/4ebb9fde47f090bcba3517952e76e846f7d13569

Reply:
iirc from reading boto3 was the automagically generated thing from the AWS spec whereas boto was done by hand